src/index.html

```diff
@@ -1,12 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
```

Comment:
Your calendar is not centered on screen. But it's also is required by the task. Try to figure out how to do it.

Reply:
Already centered ;)